doc/src/programmers-guide.md

```diff
@@ -1124,6 +1124,82 @@ NVS entries are currently stored in plaintext and are not encrypted.  Applicatio
 sensitive security information, such as account passwords, are stored in NVS storage.
 ```
 
+### Storage
+
+AtomVM provides support for mounting and unmounting storage on ESP32 devices, such as SD cards or internal flash memory. This functionality is accessible through the `esp:mount/4` and `esp:umount/1` functions.
```

Comment:
This is fantastic! Thank you. My only suggestion is the first time a module and function name are introduced in the Programmer's Guide, I try to remember to make the name a link to the module or function in the API section.

Reply:
Great point!